MIGRATION.md

```diff
@@ -43,7 +43,32 @@ In version 8 of this library, the following components have been added or remove
 ### Removed
 
 - Typography
-    - This component has been removed from the library can be replaced by [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper.
+    - This component has been removed from the library can be replaced by [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper. 
```

Comment:
```suggestion
    - This component has been removed from the library and can be replaced by the [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper. 
```

Reply:
Probably should be "These components", plural